### DISCUSSION TITLE
Trailing Space in Folder Name causes Error

### DISCUSSION THREAD

**Discussion by SteveMattan** [ttscoff/bunch-docs#175] [General]
I was creating my first Bunch and was getting an error when trying to open a folder on an external volume. I could open other folders on the volume without a problem.

It took a while but I finally determined that the issue was that the folder name had a trailing space. Once the space was removed the folder opened without issue.

As I did not see this issue in the trouble shooting section of the documentation, nor did I find it searching the forum,  I thought I would note it here.

Cheers.

**Comment by ttscoff** [comment]
So the space was in the actual folder name, or in the Bunch reference to it?

- Brett

**Reply by SteveMattan** [reply]
Brett,

The space was in the folder name, "Originals ". I must have fat fingered it when I created the folder.

I used "Get Info" and copied the file path, and then pasted it into the bunch file. I suspect that since whitespace is ignored it was not finding the full name.

Of course the blank was not obvious in the logged messages nor when looking at the the text in the bunch file. It was only when I clicked on the name, to copy it directly, that the space became obvious. 

Let me know if you'd like anymore info.

Stay well,

SteveM

**Comment by ttscoff** [comment]
That makes sense. Bunch trims whitespace from all lines. Reading whitespace literally would lead to far more errors than the type you're running into.

- Brett
